lib/elixir/lib/kernel.ex

```diff
@@ -20,9 +20,8 @@ defmodule Kernel do
   cannot be skipped. These are described in `Kernel.SpecialForms`.
 
   Some of the functions described in this module are inlined by
-  the Elixir compiler into their Erlang counterparts in the
-  [`:erlang` module](http://www.erlang.org/doc/man/erlang.html).
-  Those functions are called BIFs (built-in internal functions)
+  the Elixir compiler into their Erlang counterparts in the `:erlang`
+  module. Those functions are called BIFs (built-in internal functions)
```

Comment:
I'm not in favour of this change.
I think it's good to have a new line after a stop. That means that next time when modify the first sentence, text won't expand out of the 100 chars and we don't have to modify other lines before of this change.